test/parallel/test-regression-object-prototype.js

```diff
@@ -1,7 +0,0 @@
-'use strict';
```

Comment:
Also, that's quite interesting, I didn't expect the global object to inherit from Object.prototype.